go/grpcweb/options.go

```diff
@@ -15,6 +17,7 @@ var (
 		originFunc:                     func(origin string) bool { return false },
 		allowNonRootResources:          false,
 		corsMaxAge:                     10 * time.Minute,
+		websocketCompressionMode:       websocket.CompressionNoContextTakeover,
```

Comment:
Is this the current value?

Reply:
Yes, here is the definition https://github.com/nhooyr/websocket/blob/master/compress.go#L22
`iota` means starting from 0